src/inputs/UploadFile/UploadFile.js

```diff
@@ -109,13 +110,13 @@ export const UploadFile = (props) => {
           )}
         {(file.status === UPLOAD_FILE_STATUS_KEY.UPLOADING ||
           file.status === UPLOAD_FILE_STATUS_KEY.DOWNLOADING ||
-          file.status === UPLOAD_FILE_STATUS_KEY.DELETING) && <CircularProgress />}
-        {isFileValid === false && (
-          <FadingTooltip title={labels.invalidFileMessage} placement="right-end">
-            <ErrorIcon color="error" />
-          </FadingTooltip>
-        )}
```

Comment:
The same question about the use of `isFileValid` :slightly_smiling_face:

Reply:
Same as above.

Reply:
But if there is a web app which actually uses it, removing existing code could be a breaking change for it ? Don't we need to keep this code and just not use the prop in our own web app ?